### DISCUSSION TITLE
laravel storage with google cloud storage

### DISCUSSION THREAD

**Discussion by adamgyongyosi** [laravel/framework#50947] [Ideas] [answered]
can you add official support with google cloud storage ?

**Comment by henzeb** [comment, marked as answer]
You can of course create a PR for that, but I doubt it will get in. mean while, our Belgian friends at Spatie have a nice package for that: https://github.com/spatie/laravel-google-cloud-storage

**Reply by adamgyongyosi** [reply]
thx
